_posts/2016/2016-08-29-sageone-approach.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "approach of sageone connector as a deep integrated app"
```

Comment:
What do you think of those titles?
How we integrated Sage One into our ecommerce solution
How we connected Sage One with ePages

something like that?